lib/providers/services/http_service.dart

```diff
@@ -419,7 +419,7 @@ class HttpService {
 
     String reversedHash = reverseHash(inviteHash);
 
-    String inviteURL = "https://node.hypha.earth/v1/chain/get_table_rows";
+    String inviteURL = "https://node.hypha.earth/v1/chain/get_table_rows"; // todo: Why is this still Hypha when config has changed?
```

Comment:
our own node is normally the most reliable

we don't run hyperion yet so we don't have /v2/... endpoints though.

it's odd but ok

Reply:
Ah ok